### ISSUE TITLE
🐛 BUG: Inline styles in Astro components are not processed with `@snowpack/plugin-postcss`

### ISSUE THREAD

**Issue by jasikpark** [snowpackjs/astro#1234]
### What package manager are you using?

yarn

### What operating system are you using?

Mac

### Describe the Bug

While trying to get the container queries polyfill setup with Astro as a PostCSS plugin, I wanted to use container query syntax in the inline style tags, ex:

<details>
<summary><code>src/pages/index.astro</code></summary>

```astro
<html lang="en">

<head>
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width" />
	<title>Astro with PostCSS</title>
	<script async defer src="https://unpkg.com/cqfill"></script>
</head>

<style>
body {
	padding: 1rem;
	font-family: "Rubik";
	line-height: 1.4;
}

.test-wrapper {
	contain: layout inline-size;
	resize: horizontal;
	overflow: auto;
	margin: 1rem auto;
	outline: solid 1px grey;
	outline-offset: 4px;
}

.card {
	display: flex;
	flex-direction: column;
	flex-wrap: wrap;

	gap: 1rem;
}

.card__thumb {
	aspect-ratio: 1 / 1;
	flex: 0 0 150px;
	min-height: 150px;
	background-color: #4caf4c;
	border-radius: 7px;
}

.card__title {
	font-weight: bold;
	font-size: 1.5rem;
	margin-bottom: 0.5rem;
	text-align: center;
}

.card__content {
	flex: 1;
}

@container (min-width: 400px) {
	.card {
		flex-direction: row;
	}

	.card__thumb {
		flex: 0 0 100px;
		min-height: 100px;
		align-self: flex-start;
	}
}

@container (min-width: 600px) {
	.card__thumb {
		flex: 0 0 150px;
		min-height: 150px;
	}
}

@container (min-width: 800px) {
	.card {
		position: relative;
		justify-content: center;
		align-items: center;
		min-height: 350px;
	}

	.card__thumb {
		position: absolute;
		left: 0;
		top: 0;
		width: 100%;
		height: 100%;
		opacity: 0.25;
	}

	.card__content {
		position: relative;
		flex: unset;
	}

	.card__title {
		font-weight: bold;
		font-size: 1.5rem;
		margin-bottom: 0.5rem;
	}

	.card__desc {
		max-width: 480px;
		margin-left: auto;
		margin-right: auto;
		text-align: center;
		color: #222;
	}
}

.note {
	padding: 1rem;
	margin-top: 1rem;
	text-align: center;
}

</style>

<body>
	<h1>Astro with PostCSS</h1>

	<div class="test-wrapper" style="width: 320px">
		<div class="card">
			<div class="card__thumb"></div>
			<div class="card__content">
				<h3 class="card__title">
					Inspect Element As A Way To Feed Your Curiosity
				</h3>
				<p class="card__desc">
					One of the best gifts we have as front-end developers is inspect
					element.
				</p>
			</div>
		</div>
	</div>

	<div class="note">
		<p>This is a demo from the <a href="http://ishadeed.com/article/say-hello-to-css-container-queries/">container queries</a> article by Ahmad Shadeed</p>
	</div>
</body>

</html>
```

</details>

Which does not produce the PostCSS processed syntax, while importing the CSS `import "./global.css";` works perfectly fine in @jonathantneal's example.

### Steps to Reproduce

Compare @jonathantneal's working example: 
  - https://github.com/jonathantneal/astro-with-postcss
  - https://codesandbox.io/s/github/jonathantneal/astro-with-postcss?file=/src/styles/global.css 
to my broken fork: 
  - https://github.com/jasikpark/astro-with-postcss/tree/main/
  - https://codesandbox.io/s/github/jasikpark/astro-with-postcss/tree/main/

### Link to Minimal Reproducible Example (Optional)

https://codesandbox.io/s/github/jasikpark/astro-with-postcss/tree/main/

**Comment by jasikpark** [comment]
For sure! I suppose I'll also have to check out @FredKSchott's branch that explores going gungho on allowing full postcss integration 😆

**Comment by FredKSchott** [comment]
please do! You could probably tackle this and https://github.com/snowpackjs/astro/issues/1279 at the same time!

**Comment by FredKSchott** [comment]
Thanks for the report! We believe that this has been fixed in astro v0.21, going out today. Let us know if you can still reproduce the issue after updating to v0.21 and we'll be happy to take a look.
